lang/sk.json

```diff
@@ -35,103 +35,103 @@
+		"manyFilesWait": "Kopírovať {count} súbory.",
+		"oneFileDialogTitle": "Skopírovať súbory do...",
+		"oneFileWait": "Kopírujem súbory.",
+		"operationLabel": "Kopírujem súbory",
+		"operationSummary": "Počet skopírovaných súborov: {count}."
 	},
 	"move": {
 		"dropMenuItem": "Presunúť sem",
-		"errorDialogTitle": "[MISSING]Files that cannot be moved:",
-		"manyFilesDialogTitle": "[MISSING]Move {count} Files To...",
-		"manyFilesWait": "[MISSING]Moving {count} files.",
-		"oneFileDialogTitle": "[MISSING]Move File To...",
-		"oneFileWait": "[MISSING]Moving file.",
-		"operationLabel": "[MISSING]Moving Files",
-		"operationSummary": "[MISSING]Number of files moved: {count}."
+		"errorDialogTitle": "Súbory, ktoré sa nadajú presunúť:",
+		"manyFilesDialogTitle": "Presunúť {count} súbory do...",
+		"manyFilesWait": "Presúvam {count} súbory.",
+		"oneFileDialogTitle": "Presunúť súbor do...",
+		"oneFileWait": "Presúvam súbor.",
+		"operationLabel": "Presúvam súbory",
+		"operationSummary": "Počet presunutých súborov: {count}."
 	},
 	"upload": {
 		"addFiles": "Pridať súbory",
-		"bytesCountProgress": "[MISSING]({bytesUploaded} of {bytesTotal})",
-		"details": "[MISSING]Details",
-		"filesCountProgress": "[MISSING]{filesUploaded} of {filesTotal}",
+		"bytesCountProgress": "({bytesUploaded} z {bytesTotal})",
+		"details": "Detaily",
+		"filesCountProgress": "{filesUploaded} z {filesTotal}",
 		"progressLabel": "Prebieha kopírovanie.",
-		"progressMessage": "[MISSING]Uploading...",
+		"progressMessage": "Nahrávam...",
 		"selectFileLabel": "Vyberte súbor, ktorý chcete prekopírovať na server",
-		"selectFiles": "[MISSING]Select files to upload",
-		"success": "[MISSING]Upload finished!",
-		"summary": "[MISSING]Uploaded files: {count}"
+		"selectFiles": "Vybrať súbory na nahranie",
+		"success": "Nahrávanie dokončené!",
+		"summary": "Nahrané súbory: {count}"
 	},
 	"settings": {
 		"display": "Zobraziť:",
 		"displayDate": "Dátum",
 		"displayName": "Názov súboru",
 		"displaySize": "Veľkosť súboru",
-		"sortAscending": "[MISSING]Ascending",
-		"sortBy": "[MISSING]Sort by",
-		"sortByOrder": "[MISSING]Order",
-		"sortDescending": "[MISSING]Descending",
-		"thumbnailSize": "[MISSING]Thumbnail Size",
+		"sortAscending": "Vzostupne",
+		"sortBy": "Zoradiť podľa",
+		"sortByOrder": "Druh",
+		"sortDescending": "Zostupne",
+		"thumbnailSize": "Veĺkosť Thumbnail",
 		"title": "Nastavenia",
-		"viewType": "[MISSING]View",
-		"viewTypeList": "[MISSING]List",
-		"viewTypeThumbnails": "[MISSING]Thumbnails"
+		"viewType": "Ukážka",
+		"viewTypeList": "Zoznam",
+		"viewTypeThumbnails": "Thumbnails"
 	},
 	"errors": {
 		"codes": {
 			"10": "Neplatný príkaz.",
 			"11": "V požiadavke nebol špecifikovaný typ súboru.",
 			"12": "Nepodporovaný typ súboru.",
-			"13": "[MISSING]The connector configuration file is invalid.",
-			"14": "[MISSING]Invalid connector plugin: {pluginName}.",
+			"13": "Konfigurácia súboru je neplatná.",
```

Comment:
There seems to be a small misunderstanding here. This should really mean: "The configuration file of the CKFinder server-side connector is invalid.".